index.js

```diff
@@ -98,6 +99,43 @@ const koaV1Middleware = ({
   }
 }
 
+const hapiPlugin = ({
+  pkg: pkgJson,
```

Comment:
Do we really need to read `package.json` and pass it here? I think we should be fine with using `name` option instead of `pkg`. According to hapi docs `version` is optional and only used informatively to enable other plugins to find out the versions loaded.

Reply:
No, we don't. Anyway, as the name is taken from `package.json`, I thought we can pass it right away